lib/maintainers.nix

```diff
@@ -55,6 +55,7 @@
   antonxy = "Anton Schirg <anton.schirg@posteo.de>";
   apeschar = "Albert Peschar <albert@peschar.net>";
   apeyroux = "Alexandre Peyroux <alex@px.io>";
+  arc = "Arcadio Rubio García <arc@well.ox.ac.uk>";
```

Comment:
Do you mind naming the attribute after your GitHub username `arcadio`?

Reply:
No, it's fine.